tests/e2e/create-meta-db.js

```diff
@@ -42,27 +42,26 @@ describe('Create user meta db : ', () => {
   beforeEach(utils.beforeEach);
   afterEach(utils.afterEach);
 
-  it('should allow a new user to read/write from meta db', () => {
-    usersPage.openAddUserModal();
-    addUserModal.fillForm(userName, fullName, password);
-    addUserModal.submit();
-    helper.waitForAngularComplete();
-    commonElements.goToLoginPage();
-    loginPage.login(userName, password, false);
-    commonElements.calm();
-    helper.waitForAngularComplete();
+  it('should allow a new user to read/write from meta db', async () => {
+    await usersPage.openAddUserModal();
+    await addUserModal.fillForm(userName, fullName, password);
+    await addUserModal.submit();
+    await helper.waitForAngularComplete();
```

Comment:
I've asked a clarifying question in the angular migration channel. Might spend a few moments ripping that function out depending on the answer.

Reply:
I have removed our implementation to wait and using `browser.waitForAngular()`. Also updated some async/await since the execution changes then.  Can you review again @craig-landry